apm-agent-plugins/apm-log-shader-plugin/apm-log4j2-plugin/src/test/java/co/elastic/apm/agent/log4j2/LegacyLog4j2ShadingTest.java

```diff
@@ -53,6 +53,12 @@ public void testSimpleLogShading() throws Exception {
         super.testSimpleLogShading();
     }
 
+    @Test
```

Comment:
Why do we need to override method here ? if markers are not supported there is already a conditional on `markersSupported()` in `super.testMarkers()` implementation. If there is only a single (or a few) implementations for which the `testMarkers` is not relevant, then only overriding the method without `@Test` annotation should be enough, which have the benefit of not making like the feature is supported but test is always true.

Reply:
See javadoc for this class. Only tests annotated with the JUnit 4 annotation will run and we do want it for log4j tests, old and new